resources/testdata/data/json/dates/gregory@1/ar-EG.json

```diff
@@ -123,10 +123,250 @@
       "short": "h:mm a"
     },
     "date_time": {
-      "full": "{1} في {0}",
-      "long": "{1} في {0}",
-      "medium": "{1}, {0}",
-      "short": "{1}, {0}"
+      "style_patterns": {
+        "full": "{1} في {0}",
+        "long": "{1} في {0}",
+        "medium": "{1}, {0}",
+        "short": "{1}, {0}"
+      },
+      "available_formats": [
```

Comment:
I'd prefer this to be formatted like:

```
"available_formats": [
  ["Bh", "h B"],
  ["Bhm", "h:mm B"],
  ["Bhms", "h:mm:],
  ...
]
```

I don't see any field attributes for making something more condensed in the serialization, so I think it could only be accomplished with a custom serializer. This doesn't really seem worth it.

Reply:
I think this may be your solution - https://crates.io/crates/serde-tuple-vec-map - we already use it in ICU4X, so it wouldn't add to dependency chain.

Reply:
Oof, of course I used that to read in the CLDR data, but then forgot I could use it to re-export the data. Thanks for the tip.